CHANGELOG.md

```diff
@@ -15,6 +15,11 @@ __BEGIN_UNRELEASED__
 ### Security
 __END_UNRELEASED__
 
+## [0.21.0] - 2022-05-17
+
+### Added
+- Added `getSessionId()` method for fetching the current Heap Session ID from the underlying native SDK.
```

Comment:
In the future, this should be in the `_UNRELEASED_` section above and only move down to this when we're doing a release PR.

Reply:
tweaked the changelog, let me know if I got the format right, and if it's all good, can you merge? I don't have permission on this repo.